File: tests/fixtures/simple.go
package main

import "fmt"

const VERSION = "1.0.0"

type Point struct {
    X float64
    Y float64
}

func add(a, b int) int {
    return a + b
}

func main() {
    fmt.Println("Hello, World!")
}

type Status int

const (
    StatusOK Status = iota
    StatusError
)

var globalConfig = map[string]string{
    "debug": "false",
}

File: tests/fixtures/simple.py
#!/usr/bin/env python3

VERSION = "1.0.0"

class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y
    
    def distance(self, other):
        return ((self.x - other.x)**2 + (self.y - other.y)**2)**0.5

def add(a, b):
    return a + b

def main():
    print("Hello, World!")

class Status:
    OK = 0
    ERROR = 1

global_config = {
    "debug": False,
}

async def fetch_data(url):
    pass